### ISSUE TITLE
Support query search in web

### ISSUE THREAD

**Issue by Chaho12** [trinodb/trino-gateway#365]
I noticed that search in web history page only supports 
- queryid in exact/whole match (must match all characters)

So, i suggest we should 
- support querytext search
- support partial keyword match

**Comment by mosabua** [comment]
As column filters in the UI initially

maybe also column sorting?

**Comment by ytwp** [comment]
Good idea, but won't there be performance issues when the query log is large?

**Comment by Chaho12** [comment]
@ytwp I don't think it will be too much burder to server,
Also I just noticed that in web page, the history size used at all. 
I'll fix this
